content/reference/apis/cache.md

```diff
@@ -78,14 +78,17 @@ cache.match(request, options)
 
 - `options`: Can contain one possible property: \* `ignoreMethod` (Boolean): Consider the request method a GET regardless of its actual value.
```

Comment:
I think that this line should get a similar indentation as L112 below, what do you think?

```suggestion
- `options`: Can contain one possible property: 
  - `ignoreMethod` (Boolean): Consider the request method a GET regardless of its actual value.
```